LICENCE

```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2015 Remind101, Inc
+
+MIT License
```

Comment:
Is this the logspout license?  If not, we have used BSD2 for all of the other projects we've open sourced.